moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryStatusUpdate.js

```diff
@@ -143,6 +145,12 @@ const ProjectSummaryStatusUpdate = ({ projectId, data, refetch, classes }) => {
     setStatusUpdateAddNew(false);
   };
 
+  /**
+   * Only allow the user who wrote the status to edit it
+   */
+  const editableComment =
```

Comment:
Since this returns a boolean, the javascript convention would be to name this variable `isEditableComment` to imply the boolean type. Very minor suggestion.

https://www.robinwieruch.de/javascript-naming-conventions/